### ISSUE TITLE
lightbox-dialog: focus indicator too small

### ISSUE THREAD

**Issue by agliga** [eBay/skin#1636]
On lightbox dialog, close button. when tabbing through close button and the focus indicator shows, it is too small for the button. The focus should be on the outer part of the button, but it actually immediatly touches the X part (and it looks odd)
<img width="583" alt="Screen Shot 2022-01-12 at 9 03 13 AM" src="https://user-images.githubusercontent.com/1755269/149188764-96b0bcff-89a1-46a7-9fd3-384c06b666b8.png">
Should increase the focus indicator. 

Should check other dialogs if it has the same issue.

**Comment by edvardsniedre** [comment]
Related issues #1632, #1612
